### PR TITLE
FORMIT-10713: Plugins: update PluginUtils18_0 with new unit support

### PR DESCRIPTION
**Pull request by josh-goldstein-adsk**
- Updating PluginUtils_18 with new unit support
- Updating plugins to look at relative path for shared scripts

### CODE REVIEW

File: SharedPluginFiles/PluginUtils18_0.js [hunk 1]
```diff
@@ -5,14 +5,34 @@ if (typeof FormIt === 'undefined')
 
 FormIt.PluginUtils = {};
 
-// Function to conver the given length into the current FormIt units.
+// convert the given length into the current FormIt units
```
Comment (jhauswirth) [inline SharedPluginFiles/PluginUtils18_0.js:8]: Add comment
// This function only works on the FormIt side

File: SharedPluginFiles/PluginUtils18_0.js [hunk 2]
```diff
@@ -5,14 +5,34 @@ if (typeof FormIt === 'undefined')
...
+        case FormIt.UnitType.kMetricMeter:
+            return  length * kMetricMeterFactor;
+        case FormIt.UnitType.kMetricCentimeter:
+            return  length * kMetricCentimeterFactor;
+        case FormIt.UnitType.kMetricMillimeter:
+            return  length * kMetricMillimeterFactor;
+        // if it's none of these, we're in feet, which is the default
+        // so return the length unchanged
+        default:
+            console.log("Did we get here?");
```
Comment (jhauswirth) [inline SharedPluginFiles/PluginUtils18_0.js]: Make a better error msg

File: SharedPluginFiles/PluginUtils18_0.js (hunk 1 above)
Comment (josh-goldstein-adsk) [reply SharedPluginFiles/PluginUtils18_0.js:8]: Done.

File: SharedPluginFiles/PluginUtils18_0.js (hunk 2 above)
Comment (josh-goldstein-adsk) [reply SharedPluginFiles/PluginUtils18_0.js]: Whoops, this wasn't supposed to be there at all. Deleted.

**Pull request merged**